lib/otel/segments/server.js

```diff
@@ -69,6 +75,7 @@ function httpSegment({ agent, otelSpan, transaction, httpMethod }) {
   transaction.trace.attributes.addAttribute(DESTINATION, 'request.method', httpMethod)
   return agent.tracer.createSegment({
```

Comment:
this recorder also has to be assigned to the rpc segments.  I'm not sure about the fallback server though, probably yes because they're all web requests.